### ISSUE TITLE
qa: CoinSelection benchmark assert fails

### ISSUE THREAD

**Issue by jamesob** [bitcoin/bitcoin#13831]
```
$ ./src/bench/bench_bitcoin -filter=CoinSelection

# Benchmark, evals, iterations, total, min, max, median
bench_bitcoin: bench/coin_selection.cpp:54: void CoinSelection(benchmark::State&): Assertion `success' failed.
[1]    24547 abort (core dumped)  ./src/bench/bench_bitcoin -filter=CoinSelection

$ cat <(git rev-parse HEAD) <(uname -a) <(gcc --version)

4d550ffab64d79a44fcaa76f6e30c28f01614d3f
Linux james 4.4.0-119-generic #143+jamesob SMP Mon Apr 16 21:47:24 EDT 2018 x86_64 x86_64 x86_64 GNU/Linux
gcc (Ubuntu 5.4.0-6ubuntu1~16.04.10) 5.4.0 20160609
```

Based on some cursory debugging, [this conditional](https://github.com/bitcoin/bitcoin/blob/4d550ffab64d79a44fcaa76f6e30c28f01614d3f/src/wallet/coinselection.cpp#L78-L80) is causing the BnB routine to return false which then fails [the assert](https://github.com/bitcoin/bitcoin/blob/4d550ffab64d79a44fcaa76f6e30c28f01614d3f/src/bench/coin_selection.cpp#L52-L53).

**Comment by ken2812221** [comment]
Should be fixed by #13822

**Comment by jamesob** [comment]
Confirmed fixed. Thanks for the pointer, @ken2812221.
